src/perf/lib/Tcp.cpp

```diff
@@ -199,7 +199,7 @@ bool TcpWorker::Initialize(TcpEngine* _Engine, uint16_t PartitionIndex)
     Engine = _Engine;
     ExecutionContext.Callback = DoWork;
     ExecutionContext.Context = this;
-    ExecutionContext.Ready = TRUE;
+    InterlockedFetchAndSetBoolean(&ExecutionContext.Ready);
```

Comment:
are you just trying to WriteBooleanNoFence?

Reply:
Perhaps. I'm not sure if the fence is needed or not. We already had this helper so I was just using it.

Reply:
On Windows, at least, WriteBooleanNoFence/WriteBooleanRelease are the recommended ways to set a volatile boolean - they should suppress any ISO volatile warnings you're seeing.

Reply:
Updated